pkg/network/kubemacpool.go

```diff
@@ -110,3 +125,24 @@ func generateRandomMacPrefix() ([]byte, error) {
 
 	return prefix, nil
 }
+
+func validateRange(startMac, endMac net.HardwareAddr) error {
+	for idx := 0; idx <= 5; idx++ {
```

Comment:
Why do we skip the last octet?

Reply:
Thank you  for the notes Petr.
I think we don't skip the last octet since there are six octets, and we use "<=", right?

Reply:
:man_facepalming: sorry, you are absolutely right